tests/test_backend.py

```diff
@@ -695,6 +695,23 @@ def func(x, y):
             return tf.identity(conv, name=_TFOUTPUT)
         self._run_test_case(func, [_OUTPUT], {_INPUT: x_val, _INPUT1: x_val}, rtol=0.08)
 
+    @check_tf_min_version("2.11")
+    @check_tf_max_version("2.12")
```

Comment:
Does it mean this test only works for 2.11 and 2.12 versions? Did you try it on 2.13?

Could you please describe what kind of issues you met when tf version < 2.11?

Reply:
Actually, 2.13 works, but old TF versions don't work. PTAL again!

Reply:
Hmm, for some reason, `TF2ONNX_TEST_OPSET=18 python -m unittest test_backend.BackendTests.test_depthwise_conv3d` can pass locally after `pip install tensorflow==2.13.0`, but it's still failing on CI.

Reply:
In CI, it failed from opset 15 to opset 18, not only opset 18. Looks like the failures are relative to 2.13.0. Could you please check the tf nodes got during executing tflist_to_onnx() method to see what are differences between tensorflow 2.8.0 and 2.13.0?